src/Vlingo.Actors/Vlingo.Actors.csproj

```diff
@@ -25,10 +25,17 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="..\..\LICENSE" Pack="true" PackagePath="" />
+    <None Include="vlingo-actors.json" Pack="true" PackagePath="lib\netstandard2.0" />
```

Comment:
Why we need that ?

Reply:
Maybe i understood wrong. Dont we need to packthe default config with the package?

Reply:
Sorry. my bad didn't check that you were addressing that issue as well. Good